pkg/cmd/templatefuncs.go

```diff
@@ -28,9 +28,14 @@ type ioregData struct {
 	value map[string]any
 }
 
-// needsQuoteRx matches any string that contains non-printable characters,
-// double quotes, or a backslash.
-var needsQuoteRx = regexp.MustCompile(`[^\x21\x23-\x5b\x5d-\x7e]`)
+var (
+	// lineEndingRx matches line endings.
+	lineEndingRx = regexp.MustCompile(`(?m)(?:\r?\n)`)
```

Comment:
This should *probably* be something like `(?m)(?:\r|\r\n|\n)$` for the rare occasion that old MacOS files might be encountered (`\r` instead of either `\n` or `\r\n`).

But this will probably work regardless.

Reply:
Oh yeah, wow, you're right! PR updated.